### DISCUSSION TITLE
Mantine with Sveltejs

### DISCUSSION THREAD

**Discussion by Ali-codesweb** [mantinedev/mantine#3034] [Feature requests]
How can we use mantine library with sveltejs ?

**Comment by rtivital** [comment]
Mantine is a react components library, it cannot be used with other frameworks/libraries

**Comment by flamrdevs** [comment]
@Ali-codesweb 
I just found [SvelteUI](https://www.svelteui.org/), a Svelte library (heavily inspired by but not affiliated with [Mantine](https://mantine.dev/))
this means Mantine is an awesome UI library
